components/perf-tools/msr-safe/SPECS/msr-safe.spec

```diff
@@ -0,0 +1,133 @@
+#---------------------------------------------------------------------------bh-
+# This RPM .spec file is part of the OpenHPC project.
+#
+# It may have been modified from the default version supplied by the underlying
+# release package (if available) in order to apply patches, perform customized
+# build/install configurations, and supply additional files to support
+# desired integration conventions.
+#
+#----------------------------------------------------------------------------eh-
+
+# msr-safe.spec
+
+%include %{_sourcedir}/OHPC_macros
+%{!?PROJ_DELIM: %global PROJ_DELIM -ohpc}
```

Comment:
this is provided by the global macro file

Reply:
Remove line 14?

Reply:
Yes, the PROJ_DELIM macro is set by the global macro file.

Reply:
Done. Will be in next pull request rev.